docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
```

Comment:
you can force the user to the current one by adding the following line, and fixing permission issue for files generated by the container (uploaded data, local session, cache, logs, ...)
```
    user: 1000
```
But it may be worse, when the logged user is not the user 1000 (multiple users sharing the same machine)...
What's about commenting the line and explaining it in the documentation?..